About.md

```diff
@@ -0,0 +1,63 @@
+# About the Lattice Standard
+
+## What the Lattice Standard is
+
+1. The Lattice Standard is an *abstract* description of a lattice for the simulated transport of charged particles.
+
+1. A Lattice is an ordered list of regions in space distinguished by the presense of (possibly time-varying) specially described electromagnetic fields,
+materials, apertures and other possible engineered structures. For simplicity, we will denote these regions as _lattice_elements_ although a region
+may be distinguished by the absence of material (drift space.)
+
+1. A lattice will have the notion of a _reference_trajectory_ which is the defined path of the particle which the lattice is intended to transport.
+
+1. The positioning of lattice elements will be described by their order in the list of elements and their $s$ coordinate, the distance along the path of
+propagated reference particle. The definition and use of coordinates transverse to the direction of motion is not in this document  and will be
+the subject of the definition of the standard.
```

Comment:
I am confused by "The definition... of the standard.". The transverse coordinates are needed to define how lattice elements are oriented with respect to each other. I would leave this sentence out.